tools_docker/test_docker/README.md

```diff
@@ -0,0 +1,44 @@
+# Docker Tests
+
+This is a too to help test docker, for simple case, but also for complicated test.
+This allow:
+- to run automatically tracim, collabora and elasticsearch with by defaut a working case where all features of tracim work (it just need some tweak
+for email and you need to setup resolving using special docker)
+- to test if docker start correctly
+- to test automatically service running in the docker
```

Comment:
```suggestion
This is a tool to help test Docker images, for simple cases, but also for complicated tests.
This allows:
- automatically running Tracim with Collabora and Elasticsearch on by default (it just needs some tweaks
for emails and you need to setup resolving using a special Docker image)
- testing if the Docker container correctly starts
- automatically testing services running in the docker container
```
What is resolving?

When talking about Docker, please be clear when speaking about images, containers, etc.

Reply:
```suggestion
This is a tool to help test Docker images, for simple cases, but also for complicated tests.
This allows:
- automatically running Tracim with Collabora and Elasticsearch on by default (it just needs some tweaks
for emails and you need to setup dns resolving using a special Docker image)
- testing if the Docker container correctly starts
- automatically testing services running in the docker container
```

resolving is dns resolving of names. This permit to make all work as expected with local dns.